localstack/services/awslambda/invocation/lambda_service.py

```diff
@@ -402,3 +405,26 @@ def store_s3_bucket_archive(
     return store_lambda_archive(
         archive_file, function_name=function_name, region_name=region_name, account_id=account_id
     )
+
+
+def store_image_code(image_uri: str) -> ImageCode:
```

Comment:
```suggestion
def create_image_code(image_uri: str) -> ImageCode:
```

nit: doesn't seem to store anything :thinking:

Reply:
True!